database/insertData.js

```diff
@@ -39,33 +38,23 @@ const queries = [
                       index: 1
                       title: "First Name"
                       elementTypePluginCode: "shortText"
-                      visibilityCondition: { value: true }
                       category: QUESTION
-                      isRequired: true
```

Comment:
isn't it jsonb now ?

Reply:
Yes, it is, but single literals are actually valid JSON, and the evaluator accepts and returns them correctly, so this would be fine.

But I actually took all these out of the script, since I made `{"value": true}` the default, so not sure why you're seeing this..